2023-11-18T17:30:30.572-03:00  INFO 12688 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 12688 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T17:30:30.580-03:00  INFO 12688 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T17:30:31.667-03:00  INFO 12688 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T17:30:31.710-03:00  INFO 12688 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 28 ms. Found 0 JPA repository interfaces.
2023-11-18T17:30:32.206-03:00  INFO 12688 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T17:30:33.367-03:00  INFO 12688 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T17:30:33.382-03:00  INFO 12688 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T17:30:33.382-03:00  INFO 12688 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T17:30:34.185-03:00  INFO 12688 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T17:30:34.186-03:00  INFO 12688 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 3543 ms
2023-11-18T17:30:35.011-03:00  INFO 12688 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T17:30:35.774-03:00  INFO 12688 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T17:30:35.838-03:00  INFO 12688 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T17:30:35.855-03:00  INFO 12688 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T17:30:36.117-03:00  INFO 12688 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T17:30:36.293-03:00  INFO 12688 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T17:30:36.299-03:00  INFO 12688 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T17:30:37.264-03:00  INFO 12688 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T17:30:38.119-03:00  INFO 12688 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T17:30:38.126-03:00  INFO 12688 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T17:30:38.147-03:00  WARN 12688 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T17:30:38.148-03:00  INFO 12688 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T17:30:38.151-03:00  INFO 12688 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T17:30:38.153-03:00  INFO 12688 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T17:30:38.155-03:00  INFO 12688 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T17:30:38.168-03:00  INFO 12688 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T17:30:38.203-03:00 ERROR 12688 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:27:54.597-03:00  INFO 17020 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 17020 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:27:54.599-03:00  INFO 17020 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:27:55.812-03:00  INFO 17020 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:27:55.840-03:00  INFO 17020 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 18 ms. Found 0 JPA repository interfaces.
2023-11-18T18:27:56.166-03:00  INFO 17020 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:27:56.761-03:00  INFO 17020 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:27:56.774-03:00  INFO 17020 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:27:56.776-03:00  INFO 17020 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:27:56.925-03:00  INFO 17020 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:27:56.926-03:00  INFO 17020 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2263 ms
2023-11-18T18:27:56.959-03:00  INFO 17020 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:27:57.216-03:00  INFO 17020 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:27:57.219-03:00  INFO 17020 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:27:57.239-03:00  INFO 17020 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:27:57.395-03:00  INFO 17020 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:27:57.486-03:00  INFO 17020 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:27:57.489-03:00  INFO 17020 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:27:57.949-03:00  INFO 17020 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:27:58.506-03:00  INFO 17020 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:27:58.514-03:00  INFO 17020 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:27:58.535-03:00  WARN 17020 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:27:58.535-03:00  INFO 17020 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:27:58.537-03:00  INFO 17020 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:27:58.539-03:00  INFO 17020 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:27:58.541-03:00  INFO 17020 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:27:58.554-03:00  INFO 17020 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:27:58.585-03:00 ERROR 17020 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:29:44.308-03:00  INFO 12444 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 12444 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:29:44.311-03:00  INFO 12444 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:29:45.139-03:00  INFO 12444 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:29:45.165-03:00  INFO 12444 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 17 ms. Found 0 JPA repository interfaces.
2023-11-18T18:29:45.467-03:00  INFO 12444 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:29:46.059-03:00  INFO 12444 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:29:46.070-03:00  INFO 12444 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:29:46.071-03:00  INFO 12444 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:29:46.203-03:00  INFO 12444 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:29:46.203-03:00  INFO 12444 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1838 ms
2023-11-18T18:29:46.235-03:00  INFO 12444 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:29:46.478-03:00  INFO 12444 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:29:46.479-03:00  INFO 12444 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:29:46.499-03:00  INFO 12444 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:29:46.650-03:00  INFO 12444 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:29:46.726-03:00  INFO 12444 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:29:46.729-03:00  INFO 12444 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:29:47.062-03:00  INFO 12444 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:29:47.505-03:00  INFO 12444 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:29:47.512-03:00  INFO 12444 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:29:47.531-03:00  WARN 12444 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:29:47.532-03:00  INFO 12444 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:29:47.534-03:00  INFO 12444 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:29:47.536-03:00  INFO 12444 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:29:47.538-03:00  INFO 12444 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:29:47.549-03:00  INFO 12444 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:29:47.574-03:00 ERROR 12444 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:33:24.914-03:00  INFO 18312 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 18312 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:33:24.916-03:00  INFO 18312 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:33:25.735-03:00  INFO 18312 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:33:25.759-03:00  INFO 18312 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 15 ms. Found 0 JPA repository interfaces.
2023-11-18T18:33:26.061-03:00  INFO 18312 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:33:26.593-03:00  INFO 18312 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:33:26.603-03:00  INFO 18312 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:33:26.603-03:00  INFO 18312 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:33:26.722-03:00  INFO 18312 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:33:26.722-03:00  INFO 18312 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1752 ms
2023-11-18T18:33:26.765-03:00  INFO 18312 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:33:26.996-03:00  INFO 18312 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:33:26.998-03:00  INFO 18312 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:33:27.016-03:00  INFO 18312 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:33:27.156-03:00  INFO 18312 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:33:27.233-03:00  INFO 18312 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:33:27.236-03:00  INFO 18312 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:33:27.603-03:00  INFO 18312 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:33:28.036-03:00  INFO 18312 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:33:28.043-03:00  INFO 18312 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:33:28.069-03:00  WARN 18312 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:33:28.070-03:00  INFO 18312 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:33:28.072-03:00  INFO 18312 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:33:28.075-03:00  INFO 18312 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:33:28.076-03:00  INFO 18312 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:33:28.088-03:00  INFO 18312 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:33:28.112-03:00 ERROR 18312 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:34:26.328-03:00  INFO 5492 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 5492 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:34:26.331-03:00  INFO 5492 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:34:27.182-03:00  INFO 5492 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:34:27.205-03:00  INFO 5492 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 15 ms. Found 0 JPA repository interfaces.
2023-11-18T18:34:27.496-03:00  INFO 5492 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:34:28.002-03:00  INFO 5492 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:34:28.016-03:00  INFO 5492 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:34:28.017-03:00  INFO 5492 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:34:28.149-03:00  INFO 5492 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:34:28.150-03:00  INFO 5492 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1768 ms
2023-11-18T18:34:28.194-03:00  INFO 5492 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:34:28.435-03:00  INFO 5492 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:34:28.438-03:00  INFO 5492 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:34:28.455-03:00  INFO 5492 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:34:28.606-03:00  INFO 5492 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:34:28.679-03:00  INFO 5492 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:34:28.683-03:00  INFO 5492 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:34:29.016-03:00  INFO 5492 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:34:29.452-03:00  INFO 5492 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:34:29.460-03:00  INFO 5492 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:34:29.478-03:00  WARN 5492 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:34:29.479-03:00  INFO 5492 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:34:29.481-03:00  INFO 5492 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:34:29.483-03:00  INFO 5492 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:34:29.485-03:00  INFO 5492 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:34:29.497-03:00  INFO 5492 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:34:29.521-03:00 ERROR 5492 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:35:05.469-03:00  INFO 18044 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 18044 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:35:05.472-03:00  INFO 18044 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:35:06.661-03:00  INFO 18044 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:35:06.692-03:00  INFO 18044 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 19 ms. Found 0 JPA repository interfaces.
2023-11-18T18:35:07.025-03:00  INFO 18044 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:35:07.562-03:00  INFO 18044 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:35:07.575-03:00  INFO 18044 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:35:07.576-03:00  INFO 18044 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:35:07.716-03:00  INFO 18044 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:35:07.717-03:00  INFO 18044 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2167 ms
2023-11-18T18:35:07.753-03:00  INFO 18044 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:35:07.980-03:00  INFO 18044 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:35:07.982-03:00  INFO 18044 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:35:08.001-03:00  INFO 18044 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:35:08.145-03:00  INFO 18044 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:35:08.222-03:00  INFO 18044 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:35:08.225-03:00  INFO 18044 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:35:08.576-03:00  INFO 18044 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:35:09.018-03:00  INFO 18044 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:35:09.025-03:00  INFO 18044 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:35:09.044-03:00  WARN 18044 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:35:09.045-03:00  INFO 18044 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:35:09.048-03:00  INFO 18044 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:35:09.051-03:00  INFO 18044 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:35:09.053-03:00  INFO 18044 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:35:09.072-03:00  INFO 18044 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:35:09.099-03:00 ERROR 18044 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:38:27.692-03:00  INFO 13572 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 13572 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:38:27.694-03:00  INFO 13572 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:38:28.557-03:00  INFO 13572 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:38:28.582-03:00  INFO 13572 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 15 ms. Found 0 JPA repository interfaces.
2023-11-18T18:38:28.880-03:00  INFO 13572 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=ba6ce21c-5582-39ec-a904-f38b70d181b6
2023-11-18T18:38:29.410-03:00  INFO 13572 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:38:29.421-03:00  INFO 13572 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:38:29.422-03:00  INFO 13572 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:38:29.534-03:00  INFO 13572 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:38:29.535-03:00  INFO 13572 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1784 ms
2023-11-18T18:38:29.587-03:00  INFO 13572 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:38:29.822-03:00  INFO 13572 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:38:29.826-03:00  INFO 13572 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:38:29.846-03:00  INFO 13572 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:38:29.986-03:00  INFO 13572 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:38:30.066-03:00  INFO 13572 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:38:30.069-03:00  INFO 13572 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:38:30.412-03:00  INFO 13572 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:38:30.842-03:00  INFO 13572 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:38:30.849-03:00  INFO 13572 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:38:30.867-03:00  WARN 13572 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'avaliadorCreditoController' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoController.class]: Unsatisfied dependency expressed through constructor parameter 0: Error creating bean with name 'avaliadorCreditoService' defined in file [C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes\io\github\msavaliadorcredito\application\AvaliadorCreditoService.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
2023-11-18T18:38:30.868-03:00  INFO 13572 --- [main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:38:30.871-03:00  INFO 13572 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:38:30.874-03:00  INFO 13572 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:38:30.875-03:00  INFO 13572 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2023-11-18T18:38:30.887-03:00  INFO 13572 --- [main] .s.b.a.l.ConditionEvaluationReportLogger : 

Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2023-11-18T18:38:30.913-03:00 ERROR 13572 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in io.github.msavaliadorcredito.application.AvaliadorCreditoService required a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' that could not be found.


Action:

Consider defining a bean of type 'io.github.msavaliadorcredito.infra.ClienteResourceClient' in your configuration.

2023-11-18T18:39:48.482-03:00  INFO 17452 --- [main] i.g.m.MsavaliadorcreditoApplication      : Starting MsavaliadorcreditoApplication using Java 21.0.1 with PID 17452 (C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito\target\classes started by diogo in C:\Users\diogo\Desktop\Development\REPOSITORIES\micro-service-mensageria\msavaliadorcredito)
2023-11-18T18:39:48.485-03:00  INFO 17452 --- [main] i.g.m.MsavaliadorcreditoApplication      : No active profile set, falling back to 1 default profile: "default"
2023-11-18T18:39:49.334-03:00  INFO 17452 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2023-11-18T18:39:49.360-03:00  INFO 17452 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 16 ms. Found 0 JPA repository interfaces.
2023-11-18T18:39:49.655-03:00  INFO 17452 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=0806185f-f680-3276-a143-185d2a0d16bb
2023-11-18T18:39:49.757-03:00  INFO 17452 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'io.github.msavaliadorcredito.infra.CartoesResourceClient' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2023-11-18T18:39:49.759-03:00  INFO 17452 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'io.github.msavaliadorcredito.infra.ClienteResourceClient' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2023-11-18T18:39:50.191-03:00  INFO 17452 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2023-11-18T18:39:50.202-03:00  INFO 17452 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-18T18:39:50.202-03:00  INFO 17452 --- [main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.15]
2023-11-18T18:39:50.325-03:00  INFO 17452 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-18T18:39:50.326-03:00  INFO 17452 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1778 ms
2023-11-18T18:39:50.358-03:00  INFO 17452 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2023-11-18T18:39:50.608-03:00  INFO 17452 --- [main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:testdb user=SA
2023-11-18T18:39:50.610-03:00  INFO 17452 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2023-11-18T18:39:50.629-03:00  INFO 17452 --- [main] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:mem:testdb'
2023-11-18T18:39:50.784-03:00  INFO 17452 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2023-11-18T18:39:50.862-03:00  INFO 17452 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 6.2.13.Final
2023-11-18T18:39:50.865-03:00  INFO 17452 --- [main] org.hibernate.cfg.Environment            : HHH000406: Using bytecode reflection optimizer
2023-11-18T18:39:51.245-03:00  INFO 17452 --- [main] o.s.o.j.p.SpringPersistenceUnitInfo      : No LoadTimeWeaver setup: ignoring JPA class transformer
2023-11-18T18:39:51.681-03:00  INFO 17452 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000489: No JTA platform available (set 'hibernate.transaction.jta.platform' to enable JTA platform integration)
2023-11-18T18:39:51.688-03:00  INFO 17452 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:39:51.799-03:00  WARN 17452 --- [main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2023-11-18T18:39:51.894-03:00  INFO 17452 --- [main] o.s.c.openfeign.FeignClientFactoryBean   : For 'msclientes' URL not provided. Will try picking an instance via load-balancing.
2023-11-18T18:39:51.961-03:00  INFO 17452 --- [main] o.s.c.openfeign.FeignClientFactoryBean   : For 'mscartoes' URL not provided. Will try picking an instance via load-balancing.
2023-11-18T18:39:52.776-03:00  INFO 17452 --- [main] DiscoveryClientOptionalArgsConfiguration : Eureka HTTP Client uses RestTemplate.
2023-11-18T18:39:52.843-03:00  WARN 17452 --- [main] iguration$LoadBalancerCaffeineWarnLogger : Spring Cloud LoadBalancer is currently working with the default cache. While this cache implementation is useful for development and tests, it's recommended to use Caffeine cache in production.You can switch to using Caffeine cache, by adding it and org.springframework.cache.caffeine.CaffeineCacheManager to the classpath.
2023-11-18T18:39:52.891-03:00  WARN 17452 --- [main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.springframework.amqp.rabbit.core.RabbitAdmin
2023-11-18T18:39:52.939-03:00  INFO 17452 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 64649 (http) with context path ''
2023-11-18T18:39:52.940-03:00  INFO 17452 --- [main] .s.c.n.e.s.EurekaAutoServiceRegistration : Updating port to 64649
2023-11-18T18:39:52.947-03:00  INFO 17452 --- [main] o.s.c.n.eureka.InstanceInfoFactory       : Setting initial instance status as: STARTING
2023-11-18T18:39:52.991-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Initializing Eureka in region us-east-1
2023-11-18T18:39:53.000-03:00  INFO 17452 --- [main] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2023-11-18T18:39:53.030-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2023-11-18T18:39:53.031-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2023-11-18T18:39:53.031-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2023-11-18T18:39:53.031-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Application is null : false
2023-11-18T18:39:53.033-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2023-11-18T18:39:53.033-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Application version is -1: true
2023-11-18T18:39:53.033-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2023-11-18T18:39:53.427-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : The response status is 200
2023-11-18T18:39:53.430-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Starting heartbeat executor: renew interval is: 30
2023-11-18T18:39:53.433-03:00  INFO 17452 --- [main] c.n.discovery.InstanceInfoReplicator     : InstanceInfoReplicator onDemand update allowed rate per min is 4
2023-11-18T18:39:53.438-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1700343593437 with initial instances count: 3
2023-11-18T18:39:53.442-03:00  INFO 17452 --- [main] o.s.c.n.e.s.EurekaServiceRegistry        : Registering application MSAVALIADORCREDITO with eureka with status UP
2023-11-18T18:39:53.443-03:00  INFO 17452 --- [main] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1700343593443, current=UP, previous=STARTING]
2023-11-18T18:39:53.445-03:00  INFO 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b: registering service...
2023-11-18T18:39:53.464-03:00  INFO 17452 --- [main] i.g.m.MsavaliadorcreditoApplication      : Started MsavaliadorcreditoApplication in 5.539 seconds (process running for 6.409)
2023-11-18T18:39:53.496-03:00  INFO 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b - registration status: 204
2023-11-18T18:44:53.050-03:00  INFO 17452 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2023-11-18T18:45:18.994-03:00  INFO 17452 --- [SpringApplicationShutdownHook] o.s.c.n.e.s.EurekaServiceRegistry        : Unregistering application MSAVALIADORCREDITO with eureka with status DOWN
2023-11-18T18:45:18.995-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1700343918995, current=DOWN, previous=UP]
2023-11-18T18:45:18.995-03:00  INFO 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b: registering service...
2023-11-18T18:45:19.005-03:00  INFO 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] c.n.d.s.t.d.RedirectingEurekaHttpClient  : Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:8761/eureka/} exception=I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information stacktrace=org.springframework.web.client.ResourceAccessException: I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at org.springframework.web.client.RestTemplate.createResourceAccessException(RestTemplate.java:888)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:868)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:764)
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:646)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateEurekaHttpClient.register(RestTemplateEurekaHttpClient.java:81)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.execute(RedirectingEurekaHttpClient.java:91)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:120)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:820)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.apache.hc.client5.http.HttpHostConnectException: Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:751)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.lambda$connectSocket$0(PlainConnectionSocketFactory.java:85)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:84)
	at org.apache.hc.client5.http.socket.ConnectionSocketFactory.connectSocket(ConnectionSocketFactory.java:113)
	at org.apache.hc.client5.http.impl.io.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:181)
	at org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:447)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:162)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:172)
	at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:142)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:192)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:96)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:152)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:115)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:170)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:106)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.springframework.http.client.HttpComponentsClientHttpRequest.executeInternal(HttpComponentsClientHttpRequest.java:93)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:101)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateTransportClientFactory.lambda$restTemplate$0(RestTemplateTransportClientFactory.java:143)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:87)
	at org.springframework.http.client.InterceptingClientHttpRequest.executeInternal(InterceptingClientHttpRequest.java:71)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:862)
	... 21 more

2023-11-18T18:45:19.005-03:00  WARN 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] c.n.d.s.t.d.RetryableEurekaHttpClient    : Request execution failed with message: I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
2023-11-18T18:45:19.021-03:00  INFO 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] c.n.d.s.t.d.RedirectingEurekaHttpClient  : Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:8761/eureka/}, exception=I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information stacktrace=org.springframework.web.client.ResourceAccessException: I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at org.springframework.web.client.RestTemplate.createResourceAccessException(RestTemplate.java:888)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:868)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:764)
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:646)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateEurekaHttpClient.register(RestTemplateEurekaHttpClient.java:81)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.executeOnNewServer(RedirectingEurekaHttpClient.java:121)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.execute(RedirectingEurekaHttpClient.java:80)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:120)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:820)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.apache.hc.client5.http.HttpHostConnectException: Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:751)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.lambda$connectSocket$0(PlainConnectionSocketFactory.java:85)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:84)
	at org.apache.hc.client5.http.socket.ConnectionSocketFactory.connectSocket(ConnectionSocketFactory.java:113)
	at org.apache.hc.client5.http.impl.io.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:181)
	at org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:447)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:162)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:172)
	at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:142)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:192)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:96)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:152)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:115)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:170)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:106)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.springframework.http.client.HttpComponentsClientHttpRequest.executeInternal(HttpComponentsClientHttpRequest.java:93)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:101)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateTransportClientFactory.lambda$restTemplate$0(RestTemplateTransportClientFactory.java:143)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:87)
	at org.springframework.http.client.InterceptingClientHttpRequest.executeInternal(InterceptingClientHttpRequest.java:71)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:862)
	... 22 more

2023-11-18T18:45:19.022-03:00  WARN 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] c.n.d.s.t.d.RetryableEurekaHttpClient    : Request execution failed with message: I/O error on POST request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
2023-11-18T18:45:19.030-03:00  INFO 17452 --- [SpringApplicationShutdownHook] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2023-11-18T18:45:19.023-03:00  WARN 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b - registration failed Cannot execute request on any known server

com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:820) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-2.0.1.jar:2.0.1]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]

2023-11-18T18:45:19.031-03:00  WARN 17452 --- [DiscoveryClient-InstanceInfoReplicator-0] c.n.discovery.InstanceInfoReplicator     : There was a problem with the instance info replicator

com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:820) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101) ~[eureka-client-2.0.1.jar:2.0.1]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]

2023-11-18T18:45:19.032-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2023-11-18T18:45:19.034-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2023-11-18T18:45:19.035-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.netflix.discovery.DiscoveryClient    : Shutting down DiscoveryClient ...
2023-11-18T18:45:22.048-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.netflix.discovery.DiscoveryClient    : Unregistering ...
2023-11-18T18:45:22.094-03:00  INFO 17452 --- [SpringApplicationShutdownHook] c.n.d.s.t.d.RedirectingEurekaHttpClient  : Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:8761/eureka/}, exception=I/O error on DELETE request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information stacktrace=org.springframework.web.client.ResourceAccessException: I/O error on DELETE request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at org.springframework.web.client.RestTemplate.createResourceAccessException(RestTemplate.java:888)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:868)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:764)
	at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:646)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateEurekaHttpClient.cancel(RestTemplateEurekaHttpClient.java:91)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$2.execute(EurekaHttpClientDecorator.java:74)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.executeOnNewServer(RedirectingEurekaHttpClient.java:121)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.execute(RedirectingEurekaHttpClient.java:80)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.cancel(EurekaHttpClientDecorator.java:71)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$2.execute(EurekaHttpClientDecorator.java:74)
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:120)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.cancel(EurekaHttpClientDecorator.java:71)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$2.execute(EurekaHttpClientDecorator.java:74)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.cancel(EurekaHttpClientDecorator.java:71)
	at com.netflix.discovery.DiscoveryClient.unregister(DiscoveryClient.java:916)
	at com.netflix.discovery.DiscoveryClient.shutdown(DiscoveryClient.java:892)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeDestroyMethods(InitDestroyAnnotationBeanPostProcessor.java:415)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeDestruction(InitDestroyAnnotationBeanPostProcessor.java:239)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:191)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.run(DisposableBeanAdapter.java:184)
	at org.springframework.cloud.context.scope.GenericScope$BeanLifecycleWrapper.destroy(GenericScope.java:389)
	at org.springframework.cloud.context.scope.GenericScope.destroy(GenericScope.java:136)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:200)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:587)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:559)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1189)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:520)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1182)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1118)
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1084)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.doClose(ServletWebServerApplicationContext.java:173)
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1035)
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:145)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:114)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.apache.hc.client5.http.HttpHostConnectException: Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
	at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:751)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.lambda$connectSocket$0(PlainConnectionSocketFactory.java:85)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
	at org.apache.hc.client5.http.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:84)
	at org.apache.hc.client5.http.socket.ConnectionSocketFactory.connectSocket(ConnectionSocketFactory.java:113)
	at org.apache.hc.client5.http.impl.io.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:181)
	at org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:447)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:162)
	at org.apache.hc.client5.http.impl.classic.InternalExecRuntime.connectEndpoint(InternalExecRuntime.java:172)
	at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:142)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:192)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:96)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:152)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:115)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:170)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:106)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.springframework.http.client.HttpComponentsClientHttpRequest.executeInternal(HttpComponentsClientHttpRequest.java:93)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:101)
	at org.springframework.cloud.netflix.eureka.http.RestTemplateTransportClientFactory.lambda$restTemplate$0(RestTemplateTransportClientFactory.java:143)
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:87)
	at org.springframework.http.client.InterceptingClientHttpRequest.executeInternal(InterceptingClientHttpRequest.java:71)
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48)
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:862)
	... 38 more

2023-11-18T18:45:22.095-03:00  WARN 17452 --- [SpringApplicationShutdownHook] c.n.d.s.t.d.RetryableEurekaHttpClient    : Request execution failed with message: I/O error on DELETE request for "http://localhost:8761/eureka/apps/MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b": Connect to http://localhost:8761 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: no further information
2023-11-18T18:45:22.095-03:00 ERROR 17452 --- [SpringApplicationShutdownHook] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_MSAVALIADORCREDITO/msavaliadorcredito:92c6d908f57c774fedf0967864cce21b - de-registration failedCannot execute request on any known server

com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.cancel(EurekaHttpClientDecorator.java:71) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$2.execute(EurekaHttpClientDecorator.java:74) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.cancel(EurekaHttpClientDecorator.java:71) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.DiscoveryClient.unregister(DiscoveryClient.java:916) ~[eureka-client-2.0.1.jar:2.0.1]
	at com.netflix.discovery.DiscoveryClient.shutdown(DiscoveryClient.java:892) ~[eureka-client-2.0.1.jar:2.0.1]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMethod.invoke(InitDestroyAnnotationBeanPostProcessor.java:457) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeDestroyMethods(InitDestroyAnnotationBeanPostProcessor.java:415) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeDestruction(InitDestroyAnnotationBeanPostProcessor.java:239) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:191) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DisposableBeanAdapter.run(DisposableBeanAdapter.java:184) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.cloud.context.scope.GenericScope$BeanLifecycleWrapper.destroy(GenericScope.java:389) ~[spring-cloud-context-4.0.4.jar:4.0.4]
	at org.springframework.cloud.context.scope.GenericScope.destroy(GenericScope.java:136) ~[spring-cloud-context-4.0.4.jar:4.0.4]
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:200) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:587) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:559) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:1189) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:520) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:1182) ~[spring-beans-6.0.13.jar:6.0.13]
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1118) ~[spring-context-6.0.13.jar:6.0.13]
	at org.springframework.context.support.AbstractApplicationContext.doClose(AbstractApplicationContext.java:1084) ~[spring-context-6.0.13.jar:6.0.13]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.doClose(ServletWebServerApplicationContext.java:173) ~[spring-boot-3.1.5.jar:3.1.5]
	at org.springframework.context.support.AbstractApplicationContext.close(AbstractApplicationContext.java:1035) ~[spring-context-6.0.13.jar:6.0.13]
	at org.springframework.boot.SpringApplicationShutdownHook.closeAndWait(SpringApplicationShutdownHook.java:145) ~[spring-boot-3.1.5.jar:3.1.5]
	at java.base/java.lang.Iterable.forEach(Iterable.java:75) ~[na:na]
	at org.springframework.boot.SpringApplicationShutdownHook.run(SpringApplicationShutdownHook.java:114) ~[spring-boot-3.1.5.jar:3.1.5]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[na:na]

2023-11-18T18:45:22.110-03:00  INFO 17452 --- [SpringApplicationShutdownHook] com.netflix.discovery.DiscoveryClient    : Completed shut down of DiscoveryClient
